app/controllers/keywords_controller.rb

```diff
@@ -33,9 +33,33 @@ def create
     redirect_to(keywords_path)
   end
 
+  # rubocop:disable Metrics/MethodLength
+  def search
+    result = ScrapeResultSearchQuery.new(
+      pattern: search_params[:search],
+      query_type: search_params[:query_type]
+    ).call
+
+    render(locals: {
+             search_result_presenter: SearchResultPresenter.new(search_result: result),
+             search_params: search_params
+           })
+  rescue ActiveModel::ValidationError => error
+    flash[:alert] = error.model.errors.full_messages
+    redirect_to(keywords_path)
+  end
+  # rubocop:enable Metrics/MethodLength
```

Comment:
Disabling Rubcop each time is not great 😅 It might be better to revisit the settings (including for rspect) a bit as this method is not long

Reply:
well, the reason I disable Rubocop is this method is actually not long. It exceeds the threshold due to the way I format it. 

IMO, this is a valid case to disable the flag. The code path is linear (no branching) and there're no parts worth extracting out.

Reply:
The issue is that there are numerous such disabling in the codebase 😅 I agree that the code is not too long or complex so it would be best to check if the current limit makes sense for this project.

Reply:
Totally agree on that. Let me revise the rubocop configuration